go.mod

```diff
@@ -38,6 +38,6 @@ require (
 	gopkg.in/yaml.v3 v3.0.1 // indirect
 )
 
-go 1.22
+go 1.22.0
```

Comment:
I think this shouldn't be there?

Reply:
There is no need to prevent bugfix for this version of go.
Just learned today: https://go.dev/ref/mod#glos-patch-version